Source/WebCore/style/AnchorPositionEvaluator.cpp

```diff
@@ -408,7 +408,8 @@ Length AnchorPositionEvaluator::resolveAnchorValue(const BuilderState& builderSt
     String anchorString = anchorValue.anchorElementString();
     if (anchorString.isNull())
         anchorString = builderState.style().positionAnchor();
-    RefPtr anchorElement = anchorPositionedState.anchorElements.get(anchorString);
+
+    RefPtr anchorElement = !anchorString.isNull() ? anchorPositionedState.anchorElements.get(anchorString) : nullptr;
```

Comment:
Usually this kind of thing reads better without a `!` but I suppose it's a matter of taste.